src/addons/dragAndDrop/styles.less

```diff
@@ -5,7 +5,7 @@
     pointer-events: none;
 
     & .rbc-show-more,
-    & .rbc-event:not(.rbc-addons-dnd-dragging) {
+    & .rbc-event {
       pointer-events: all;
```

Comment:
i'm not sure how this would work...It seems like this would prevent dropping the event at all because drop target is always obscured by the event being dragged, preventing it from detecting `isOver`

Reply:
It's not seen by this commit but there is another style that allows this to work on line 31.

```
  &.rbc-addons-dnd-is-dragging .rbc-event {
    pointer-events: none;
    opacity: .50;
}
```

Few things going on here. I think the dragging style is now called `rbc-addons-dnd-is-dragging` instead of `rbc-addons-dnd-dragging`. Second, `rbc-event` does not get the the `rbc-addons-dnd-is-dragging` class. That class in added to the top level div along with `rbc-calendar rbc-addons-dnd`. So I think line 31 handles things correctly.

Reply:
👍 great LGTM